x/pairing/keeper/epoch_cu.go

```diff
@@ -0,0 +1,201 @@
+package keeper
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/utils"
+	"github.com/lavanet/lava/x/pairing/types"
+)
+
+// UniqueEpochSession is used to detect double spend attacks
+// It's kept in a epoch-prefixed store with a unique index: provider, project ID, chain ID and session ID
+//
+// ProviderEpochCu is used to track the CU of a specific provider in a specific epoch for unresponsiveness
+// It's kept in a epoch-prefixed store with a unique index: provider address
+//
+// ProviderConsumerEpochCu is used to track the CU between a specific provider and
+// consumer in a specific epoch for payments
+// It's kept in a epoch-prefixed store with a unique index: provider and project ID
+
+/* ########## UniqueEpochSession ############ */
+
+// SetUniqueEpochSession sets a UniqueEpochSession in the store
+func (k Keeper) SetUniqueEpochSession(ctx sdk.Context, epoch uint64, provider string, project string, chainID string, sessionID uint64) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.UniqueEpochSessionKeyPrefix(epoch))
+	store.Set(types.UniqueEpochSessionKey(provider, project, chainID, sessionID), []byte{0})
+}
+
+// GetUniqueEpochSession checks if a UniqueEpochSession exists
+func (k Keeper) GetUniqueEpochSession(ctx sdk.Context, epoch uint64, provider string, project string, chainID string, sessionID uint64) bool {
```

Comment:
this is not a get but an exist method, change the name

Reply:
done. see https://github.com/lavanet/lava/commit/e861f77d0fc4923a3dd2cef52dab81d959198321